pkg/reportingplugins/mercury/types.go

```diff
@@ -41,3 +41,16 @@ type Transmitter interface {
 	// - FromAccount() should return CSA public key
 	ocrtypes.ContractTransmitter
 }
+
+type ChainReader interface {
+	// Head returns the current chain head
+	Head(context.Context) (Block, error)
```

Comment:
This method seems redundant.

Reply:
Yes. Calling LatestBlocks(1) is enough. Left it as a placeholder as but it can be removed.